bin/polymer.js

```diff
@@ -26,5 +29,9 @@ resolve('polymer-cli', {basedir: process.cwd()}, function(error, path) {
   let lib = path ? require(path) : require('..');
   let args = process.argv.slice(2);
   let cli = new lib.PolymerCli(args);
-  cli.run();
+  cli.run().catch(err => {
+    logger.error(err);
```

Comment:
Can you include a message here? It's helpful in debugging to know where the error was caught & logged. See polymer-cli.ts `uncaughtRejection` handler for example.